libpod/container.go

```diff
@@ -203,6 +205,63 @@ func (c *Container) Config() *ContainerConfig {
 	return returnConfig
 }
 
+// RootFsSize gets the size of the container's root filesystem
+// A container FS is split into two parts.  The first is the top layer, a
+// mutable layer, and the rest is the RootFS: the set of immutable layers
+// that make up the image on which the container is based
```

Comment:
nit of a nit, missing period at end of line.

Reply:
fixed